custom_profile.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<dds xmlns="http://www.eprosima.com">
+    <profiles>
+        <transport_descriptors>
+            <transport_descriptor>
+                <transport_id>udp_transport</transport_id>
+                <type>UDPv4</type>
+            </transport_descriptor>
+            <transport_descriptor>
+                <transport_id>shm_transport</transport_id>
```

Comment:
Huh. Didn't realized shared memory works across the python publishing but cool

Reply:
I don't think it does, but the image processing pipeline is all c++ components so that is sped up

Reply:
FWIW you do get shared memory transport in Python too, but you can't escape the conversion from Python messages to C messages and back.